input/core/src/commonMain/kotlin/majestic/editor/body/chunksUI/Chunks.kt

```diff
@@ -28,8 +28,8 @@ fun Chunks(
     actions: @Composable (chunk: Chunk) -> Unit,
     labels: Labels,
     resource: Painter,
-    onDrop: () -> Unit,
-    onClick: () -> Unit
+    onImagePick: (Image) -> Unit,
+    imagePreview: @Composable (Image) -> Unit
```

Comment:
You don't have to expose this. It is fine for it to be an internal implementation of the editor